config/command/ihavereadtherules.json

```diff
@@ -2,7 +2,7 @@
     "ruleText": [
         "Hello, welcome to teh Discrodz. This is a Christian Minecraft Server.",
         "We believe in all things American like Jesus, Apple Pie and Lime Bud Light.",
-        "If you have read the rules of and wish to accept Jesus into your heart, send `>ihavereadtherules {{randomNumber}}` to auth."
+        "If you have read the rules of and wish to accept Jesus into your heart, send `!ihavereadtherules {{randomNumber}}` to auth."
```

Comment:
If we are going to have more static text in the future, perhaps we should start using **{{botClient.prefix}}**